csharp/ql/src/Security Features/CWE-327/InsecureSQLConnectionInitializerBad.cs

```diff
@@ -0,0 +1,7 @@
+using System.Data.SqlClient;
+
+// BAD, Encrypt not specified
+string connectString =
+    "Server=1.2.3.4;Database=Anything;Integrated Security=true;";
+var builder = new SqlConnectionStringBuilder(SetToTrueConnStr) { Encrypt = false }
```

Comment:
I think this is what you mean, right?
```suggestion
var builder = new SqlConnectionStringBuilder(connectString) { Encrypt = false }
```